packages/management/addons-manager-v2/bundle/config/overlays/feature-gate-clusterbootstrap.yaml

```diff
@@ -0,0 +1,12 @@
+#@ load("@ytt:data", "data")
+#@ load("@ytt:overlay", "overlay")
+
+#@ antreaconfigscrd = overlay.subset({"kind": "CustomResourceDefinition", "metadata": {"name": "antreaconfigs.cni.tanzu.vmware.com"}})
+#@ calicoconfigscrd = overlay.subset({"kind": "CustomResourceDefinition", "metadata": {"name": "calicoconfigs.cni.tanzu.vmware.com"}})
```

Comment:
**question**: We havent added CPI to the package ?

Reply:
Don't see it. I'll add it and CPI controller as well.